src/eko/couplings.py

```diff
@@ -380,10 +435,13 @@ def __init__(
             raise NotImplementedError("a_s beyond N3LO is not implemented")
         if order[1] not in [0, 1, 2]:
             raise NotImplementedError("a_em beyond NLO is not implemented")
+        if order[0] == 0:
```

Comment:
why do we need this?

Reply:
I added it since we passed from PTO = 0, 1, 2 to order[0] = 1, 2, 3 in which the value order[0] = 0 now in meaningless (it would be no evolution)

Reply:
I added it since we passed from PTO = 0, 1, 2 to order[0] = 1, 2, 3 in which the value order[0] = 0 now in meaningless (it would be no evolution)

Reply:
I see then the fix is to remove the 0 in line 434 ( `if order[0] not in [0, 1, 2, 3, 4]: ` ) and remove also this check.

Reply:
just to be sure: can we run eko with QED only using only the API? i.e. not via theory card? I think this is something we want